src/Module.php

```diff
@@ -293,10 +293,10 @@ protected function resetGlobalSettings()
      * Gets toolbar HTML
      * @since 2.0.7
      */
-    public function getToolbarHtml()
+    public function getToolbarHtml($tag)
```

Comment:
Why?

Reply:
Few week ago it was only working solution I foud, but now working again. I will revert it.